index.html

```diff
@@ -1,12 +1,24 @@
cFd1HZnhugcQB15rmkzIxrQLakdTdBpYvNVhzcopkOnmgPYYu8qXiiRdUqjpEhJViNDukpNqtm40P7CSfUy6tjrCfe7L6j1HVL4gZgA2+fT96IBZjzY+fkla1QQDvB+RMhMYoRYco8uqXq050Gg+wT3wBYdkmSucTTF+abwzGkfsftky+mCP353UhHw9HVPPpwAR5oopgD9/f0rWJENk8vO/dIyDzncDHhbfoSNPkq4psBMNv5XUmg0meosnmVTB5xKryQvTjSwXkNPkCu/8kaMGZxtmNwO59AuGAuEls94XWaOQ6JWmIynaJlxMk9dyvVcEoFtPvcn7KBgaeZzQdzfsLlWsbjnZclNvYDQKOrh5b46x+LYNbxtsEqx7HXy5ip7MI6ZqO7AepTxMNhohLmW+07k8ldPbNpAHJq4GVug+aDTrSsxBsrnMTtHdUQTUW6BkLl7LqyR3YosDjUjKBOZot2Ud1JlaqypnAY33WzB7rXEadarDG3FP39geiI59PIQ+nBA90DxHtCaDPEcblBay7yCR5JJlAsY0Od46l3E7Dkk202s/wDcyPDhOVpkyNpR+C1H4nE0mPZ4c4LjsGtBcR9PqlhvfUqJYxrDqGtB+QS+IaqVcS6UtVpbKZVpFRs23+4Q2UMoLhy358/knnsGh1/boFRhiJ108jCYSnsuSR+FxlFwbGP+E4+lre/LdJ1LBxB007pAphZEgymi76fv5UTE4wtAA13XDOKnQhMlyo60fvzKUxDi8gD8X6Ka/ik2Ep7AYiS29ydNbDdKiGqdODpbToj0aOo5hZudCO2tgmsMBKSg6DzEb/ZHptv1WqTBOXfcpunT3hLTw3gzkuR0lMOfaW36IbCGjUjY8pRHUd2mD9D+E5EhOcHtjf7FDwlfVrrOGq7qsJ8bR4x7zef+0DE0c4D2e+Nv7DkeqoN4luRwdsfoj2c1cYau2qzKexG4P5QsKSx5Y7yPMc0aMF4ablvJOVad0qxmWq07OkeqqVGqk18+4+KmGxAez/46nvQCYO/ZAo1Pb1CWPAyiLi/Ve9eGn3gCNwRMhRON/wAPkZ8McjuQMA+eyUovKC/iVnMfBc0xYXPZWv4XQe15c9oEMcQd5cQL+UqM3g+QyA72jYJDt+d916v+N1g9jyAQQ4NM8wJ9Qi/Qn2oHFtBglduqA7pTEU2H3mg97pd2G3YS3psspbGuSma9MG+6QxrHeEj4TcLl9ao3Vhd1b/tDfiuYI72T+UHxrsuDhI7JKtT1EQt08SyZBBB1XdUjY2Oh2RLKm82ION4dmBgW/dF5+rTyEjcGPmV7JzXDSd+fLkpmPwrHvbmEOJ2sbAm45aIlws1DoYZzzbQct1cwGCymXev3TVFoaA1rWgJqkzolbpyY6pskph7Mvcj5D/a0XBjcxgd4A+q4Zig7QF3MgWPmdkWqkp9jALlbZWaXZZuPoUCriMrczrbAakk6JN3he14+LXuP9fZEFW3DM0g8oPol+F40yWP1Bg/lHpO06hTOIUi13tG7Wd22Kol+o3Rw1H1HJJ1n5D7RvuH3hyPPtzRcDXztCx/hvEsdZw5H8FBYXxFK/taevxNHxDmOv3RXt9qwOb7wu0+h6FADTRcIM03Gx/qf6npyTDvA4Ob7jjfo78H7pwVtj8wadw4SDsZgqs7RScSMrsw0dr3GhVSi+wKtJRwVPhdaWZTtby2UIVbpnh1U5iOY+yjVYp47AMeDI8xqp2GwYotLRuS709FRZitktjXy7yRfopPS+TddZOZXErThKhbb6wFmiSk6uED7vuf6j3R+UyLaBdZSnh7n0j4jhoNm69Ep/wClFgMvPWCQAvSNYhVqQdYiyVhfJ5CpSLpyPc4DnIB7c0qcGZndelx+B5W7WUt7nNsb+Q+6m+KnoTGP3iP+ofhdPx4b4WgOdyGg7lAqPe+0kDom8DhGt2QMgdHBPqOzvM8hsOwVJz2U7au2A1/0ELE1yPCyx58uyFh6AHfcnUpyFazFS5su1kQOS3izDWf9/oUR4zEAbXS3FH+OmzuT9h6pktUHeEHqER4GaDo4Jeh7hRniRbWJHcKoQGEmm8s21HZPVqwa8T7rx5TuPklazc7Q4e839K6q0zUpED3hBb3H6R5pUzLWgeB12O0n7d1lNkTTfcEWPMfkIXDa2dhY6x+oITjGF7cps9pt3/BThUu5hLHMPvN37XBTuAfmY09EB+zv/F373+61w4xmZycY7G/5WiE1tSEzgakPSFUwi4R/iBWVaLYeC4LjEulyXafEey7c+6f6T+3YW1wHIiDaAXSxoXRQTQatwsBXQKAA+nKlYzBToFbLVwWJYcrzP+CWrbjFlfq0QlH4UI+J/JLYz5rbnxYJ19DkhMwqMLXFBm6m1Rnqz8uyr1mw2Bqf0oeEw15TkGmGNhjuyIWkBpG32RajIYUyKXhCchWlqbQDI0d913Qp5XkbG4/CO2iLjY/RY5hgHdv2TwaSr08j845w8fZ3oVSfs8dndufkhVAJE+64Qf35reCOQmm64+Endp/GiMGiYhkiRo7Xodik6L4e087HuP0p5gyyw6bdR+QpuIBa4g9HD7FOJqfihCzCAzYT6LvHt+qq/wAcpeEmBrGvms/tduQAAiCRqLdYXD3QQU9xtpZknm70/KnPfKeCXfRmvTTCpWeE9hKkhB04uHFbJQ3oS7aUdoS1MptqDZC4eF2UJyCcErhy7cuHBMAlqz2aMGrAmRQ0N0WlTCYLVpAcvbMDqEy9sBBotkyu6zrwgCM0WErU2XDnJk26mCI226LipTkAfELtPoth62XoDGOztg2e36H8IGJp52zEPbNvuF1VJBzD3hr/ANQ/K6e6QHt5XjcflBotb3Qqv8c916xYsufs+vpn8h92l/5eimDZYsVDn6acmOH6eaxYhSkFp60sQTmmmmLFiBXVRCcsWJkGdVgWLEBpbCxYmTGrHrSxICU1oe8sWKiFchPW1iAEdFt2ixYgND3h2QsF8X/c77lYsRQ//9k="
```

Comment:
Why is this image source so long?

Reply:
Should've used a link shortener...

Reply:
It's a base64 data encoded url. The url contains the data of the image. @timurbuch How did you get there? 😄